case_studies/weak_lensing/lensing_dc2.py

```diff
@@ -0,0 +1,190 @@
+import logging
+import sys
+
+import pandas as pd
+import torch
+
+from bliss.surveys.dc2 import DC2DataModule, DC2FullCatalog, wcs_from_wcs_header_str
+
+
+class LensingDC2DataModule(DC2DataModule):
+    def __init__(
+        self,
+        dc2_image_dir: str,
+        dc2_cat_path: str,
+        image_slen: int,  # assume square images: image_slen x image_slen
+        tile_slen: int,
+        splits: str,
+        batch_size: int,
+        num_workers: int,
+        cached_data_path: str,
+        **kwargs,
+    ):
+        super().__init__(
```

Comment:
Can you pass `*args` and `**kwargs` to the parent class, instead of explicitly listing all of them?

Reply:
they are listed since some of them are hardcoded as dummy values: i.e. min_flux_for_loss, max_sources_per_tile, image_lim, prepare_data_processes, data_in_one_cached_file, train_transforms, nontrain_transforms. 

I can instead add these hardcoded values to kwargs but this provides more visibility into the subclassing imo